### ISSUE TITLE
Add a "tags" array for links

### ISSUE THREAD

**Issue by waldyrious** [Fyrd/caniuse#160]
Currently the links array contains only the link title and url. The tags ("info", "reference", "demo", etc.) are, I suppose, added by hand during updates. It would be nice if these could also be specified in the json data files.

**Comment by CrazyPython** [comment]
This has not been commented on for two years - I suggest closing it or reconsidering it.

**Comment by waldyrious** [comment]
@CrazyPython I've seen many people closing stale issues and PRs on github lately (perhaps out of frustration from unresponsive repo owners), and honestly don't see the rationale. Either the issue is deliberately closed (wontfix, invalid, duplicate, etc) or it should remain open to track the topic. As for reconsidering it, I'd be happy to hear @Fyrd's thoughts on how this should move forward.

**Comment by CrazyPython** [comment]
He's active since his last contribution was just two days ago. Just as a reminder to get it in his notifications inbox.

**Comment by Fyrd** [comment]
Yeah, I should probably include these. Just kinda low priority.

Generally when I leave issues open they're still open for consideration.

**Comment by waldyrious** [comment]
> Generally when I leave issues open they're still open for consideration.

Exactly, I always understood that to be the standard. It's simply counter-productive to close an issue that's still valid just because the repo owners aren't responding to it. That sounds like a protest action rather than acting in the best interests of the project. 

@Fyrd, any suggestions on how one could go about doing that? Perhaps a PR would help advance this issue.
